docs/en/signal-hub-endpoint.rst

```diff
@@ -0,0 +1,169 @@
+  - notify the Credential Issuer of the availability of the attributes related to a specific Digital Credential which a User requested in its Wallet. 
+  
+The last case, referred to as deferred issuance, happens when the Credential Issuer has requested a Digital Credential's attributes from the Authentic Source (invoking the :ref:`authentic-source-endpoint:Get Attribute Claims` PDND endpoint) and the Authentic Source cannot respond immediately with the requested attributes. Thus, the Authentic Source notifies the Credential Issuer via the Signal Hub at a later time that the requested attributes are now available.
+
+The Signal Collection e-Service endpoint is used by Authentic Sources to deposit Signals to the Signal Hub via a Signal Collection request. The latter MUST be a POST request with ``Content-Type`` set to ``application/json``, whose header MUST have the parameters described in `Signal Hub Guide`_ and whose body MUST contain the following parameters:
+
+.. _table_Signal_deposit_request_parameters:
+.. list-table::
+  :widths: 25 75
+  :header-rows: 1
+
+  * - **Parameter Name**
+    - **Description**
+  * - **signalId**
+    - REQUIRED. A positive 64 bit integer number referencing the identifier of the Signal in chronological order.
+  * - **objectType**
+    - REQUIRED. This filed is a free field which the Authentic Source MAY use to further specify the Signal.
+  * - **objectId**
+    - REQUIRED. The subject to which the Signal is bound. If the Signal has ``signalType``:
+    
+      - ``CREATE``, then it MUST be set to the ``jti`` value the Credental Issuer used in the Agid-JWT-Signature token of the `get attributes` request to the Authentic Source to obtain the attributes related to a specific Digital Credential (see :ref:`authentic-source-endpoint:Get Attribute Claims`);
+      - ``UPDATE``, then it MUST be set to the Authentic Source's unique database identifier of the Digital Credential's attributes the Signal refers to.
+      
+  * - **signalType**
+    - REQUIRED. Signal Type. It MUST be one of the following: 
+
+      - ``UPDATE``, when the Signal refers to a change in the status and/or value of a specific attribute associated with a Digital Credential;
+      - ``CREATE``, when the Signal refers to the availability of the attributes of a specific Digital Credential;
+
+  * - **eserviceId**
+    - REQUIRED. e-Service to which the Signal is bound. It MUST correspond to the e-Service Id value the Authentic Source is a Provider of.
+
+.. note::
+  In the deffered issuance flow, i.e., when the Authentic Source notifies the Credential Issuer of the availability of the Digital Credential's attributes via Signal Hub; both entities MUST keep track of the Credential Issuer's ``jti`` value used in the Agid-JWT-Signature of the ``get attributes`` request. This is necessary since the ``objectId`` of the Signal MUST be set to that ``jti`` value when the Signal has ``signalType`` ``CREATE``.
```

Comment:
```suggestion
  In the deffered issuance flow, i.e., when the Authentic Source notifies the Credential Issuer of the availability of the Digital Credential's attributes via Signal Hub; both entities MUST keep track of the Credential Issuer's ``jti`` value used in the Agid-JWT-Signature of the ``get attributes`` request. This is necessary since the ``objectId`` of the Signal MUST be set to that ``jti`` value when the Signal has ``signalType`` set with ``CREATE``.
```